.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,7 @@
+---
+name  : Ask a Question
+about : The issue tracker is not for questions -- please ask questions at https://community.rstudio.com.
+---
+
+The issue tracker is not for questions. If you have a question, please feel free to ask it on our community site, at https://community.rstudio.com.
```

Comment:
Link should be to the IDE section of the community site: https://community.rstudio.com/c/rstudio-ide